frontend/src/components/Button.module.scss

```diff
@@ -0,0 +1,47 @@
+@import "~@mozilla-protocol/core/protocol/css/includes/tokens/dist/index.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/mixins/typography";
+@import "~@mozilla-protocol/core/protocol/css/includes/forms/lib";
+
+.button {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  background-color: $color-blue-50;
+  padding: $spacing-sm $spacing-2xl;
+  border-radius: $border-radius-sm;
+  border: 2px solid transparent;
+  color: $color-white;
+  font-weight: bold;
+  line-height: 1.25;
+  @include font-size(16px);
+
+  &:hover:not(:disabled) {
+    background-color: $color-blue-60;
+  }
+
+  &:focus {
+    border-color: $button-border-color-focus;
+    box-shadow: $field-focus-ring;
+    color: $color-white;
+    outline: none;
+  }
+
+  &:disabled {
+    opacity: 0.6;
+    cursor: not-allowed;
+    background-color: $color-light-gray-80;
+  }
+
+  img {
+    margin: 0 $spacing-xs;
+  }
+
+  &.isDestructive {
```

Comment:
I don't have a strong preference either way. I didn't put much consideration in this; think I went with this just out of habit and context (since classes are now mostly used in a JS context, where we do camel case). I think everything in `/frontend` is camel case, but I'm OK with spending an hour or two (or possibly finding a script to do it for me) converting that to kebab-case. Especially now that our class names are explicitly imported in our JS, that should be pretty safe: any references to classes that no longer exist should be automatically pointed out in the build.

Reply:
Let's use it kebab-case then, to match Protocol's naming system/structure. I still want that to be the default for patterns where it is possible.

Reply:
I started converting some to kebab case, but one thing I didn't realise before: since we're importing class names as properties on a `styles` object, and `-` isn't a valid character when using dot notation to access object properties, that means we'd have to use

```js
styles["is-destructive"]
```

instead of

```js
styles.is-destructive
```

or the current

```js
styles.isDestructive
```

I'm thinking all those brackets might get a bit noisy. That gives us at least three options:

1. Suck it up and just use bracket notation.
2. Switch it over to `snake_case`.
3. Keep it as-is.

Which would you prefer?

Reply:
As discussed on Slack, we're going with option 1. Changed in 10821af3.